app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -0,0 +1,146 @@
+      final membership = myMembership.value!;
+      if (membership.canString('CanInvite')) {
+        topMenu.add(
+          IconButton(
+            icon: Icon(
+              Atlas.plus_circle_thin,
+              color: Theme.of(context).colorScheme.neutral5,
+            ),
+            iconSize: 28,
+            color: Theme.of(context).colorScheme.surface,
+            onPressed: () => context.pushNamed(
+              Routes.spaceInvite.name,
+              pathParameters: {'spaceId': room.getRoomIdStr()},
+            ),
+          ),
+        );
+      }
+    }
+
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          SliverAppBar(
+            elevation: 0.0,
+            actions: <Widget>[
+              PopupMenuButton<int>(
+                icon: const Icon(Atlas.dots_vertical, size: 22),
+                itemBuilder: (context) => [
+                  PopupMenuItem<int>(
+                    value: 0,
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        const Text('Leave Room'),
+                        const SizedBox(width: 50),
+                        Icon(
+                          Icons.exit_to_app_outlined,
+                          color: Theme.of(context).colorScheme.errorContainer,
+                        )
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+            ],
+          ),
+          SliverToBoxAdapter(
+            child: Consumer(
+              builder: (context, ref, child) {
+                final convoProfile = ref.watch(chatProfileDataProvider(room));
+                final roomId = room.getRoomIdStr();
+                return convoProfile.when(
+                  data: (profile) => Column(
+                    children: [
+                      Center(
+                        child: Padding(
+                          padding: const EdgeInsets.only(
+                            top: 38,
+                            bottom: 12,
+                          ),
+                          child: SpaceParentBadge(
+                            badgeSize: 20,
+                            spaceId: room.getRoomIdStr(),
+                            child: ActerAvatar(
+                              mode: DisplayMode.GroupChat,
+                              uniqueId: roomId,
+                              displayName: profile.displayName ?? roomId,
+                              avatar: profile.getAvatarImage(),
+                              size: 75,
+                            ),
+                          ),
+                        ),
+                      ),
+                      Text(
+                        profile.displayName ?? roomId,
+                        overflow: TextOverflow.clip,
+                        style: Theme.of(context).textTheme.titleLarge,
+                      ),
```

Comment:
may I suggest we _always_ show the roomId somewhere (whether we have a display name or not) and analogues to the UserProfile have a quick-copy button next, too it?

Reply:
Do you want to have it enabled like tooltip (just as for acter-avatars)?

Reply:
no, we don't have tooltips on touch-devices, important information can never be put behind that. Just another entry below, same as for user-profile. And either hide the display name if not given or use the same pattern as the user-profile of allowing to edit an empty one if it isn't there yet. But as said, we can also push that into the another PR later.